src/Service/VendorService/OverDrive/Api/Exception/AuthException.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Should be placed in `App\Exception` with the other custom application exceptions. Maybe renamed to `VendorAuthException`

Reply:
Won't fix here